handsontable/src/editors/dateEditor/dateEditor.js

```diff
@@ -9,6 +9,7 @@ import { isFunctionKey } from '../../helpers/unicode';
 import 'pikaday/css/pikaday.css';
 
 export const EDITOR_TYPE = 'date';
+const group = 'closingDateEditor';
```

Comment:
The `group` variable name is a little generic; could you change it to be more descriptive?

Reply:
Changed within: https://github.com/handsontable/handsontable/pull/9878/commits/45ec21784bddcf2b3491b106e7e882faf539f2ca